src/main/java/br/com/sbk/sbking/gui/elements/SpecificDirectionBoardElements.java

```diff
@@ -11,36 +11,40 @@
 
 public class SpecificDirectionBoardElements {
```

Comment:
For some strange reasons, the indentation is f-d up in this file.. In my computer it is the same, don't know why. If the only modification is on the `FrameConstants.rulesetPosition);`, then it is ok. If possible, undo this indentation please, if not we'll just let it go for now and fix all files later.